### ISSUE TITLE
parallel nodeSetup-execution breaks build

### ISSUE THREAD

**Issue by trampi** [srs/gradle-node-plugin#83]
Hi there,

we use the gradle-node-plugin for our builds to have project specific node versions.
However, in our jenkins CI-Environment builds may break under the condition, that two or more projects with the same node version are built at the same time.
The nodeSetup-Task fails, because the node runtime is being downloaded twice.

```
:cms-client:nodeSetup FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':cms-client:nodeSetup'.
> Unable to expand TAR '/data/var/lib/jenkins/.gradle/caches/modules-2/files-2.1/org.nodejs/node/0.12.7/a861a6ded50c8cda32ac39c3a54699c767742e6/node-0.12.7-linux-x64.tar.gz'
    The tar might be corrupted or it is compressed in an unexpected way.
```

Any idea how to fix the problem so that we can run our builds parallel?

**Comment by dcoraboeuf** [comment]
Same than #91

**Comment by srs** [comment]
Fixed in 0.12-SNAPSHOT.
